metron-interface/metron-alerts-host/pom.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Our xml files generally follow the formatting style of 4 character indents.  Reformatting should be easy and might as well do it now.

Reply:
I'm not even sure this is accurate according to our style guide but almost every xml file follows this.

Reply:
Done, and agreed, it should be a formal style guide addition, and part of the formatting configs on the contributor wiki really.